DMX.Portal.web/Startup.cs

```diff
@@ -53,5 +53,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 endpoints.MapFallbackToPage("/_Host");
             });
         }
+
+        private static void AddBrokers(IServiceCollection services)

```

Comment:
Even though we forsee a need to add more services, just to keep it clean every single time consider => as it is a single line of code.